probe/host/controls.go

```diff
@@ -0,0 +1,66 @@
+package host
+
+import (
+	"os/exec"
+
+	log "github.com/Sirupsen/logrus"
+	"github.com/kr/pty"
+
+	"github.com/weaveworks/scope/common/xfer"
+	"github.com/weaveworks/scope/probe/controls"
+)
+
+// Control IDs used by the host integration.
+const (
+	ExecHost = "host_exec"
+)
+
+// Controls handles controls for a hosts.
+type Controls struct {
+	pipes controls.PipeClient
+}
```

Comment:
Actually, in docker, the controls are associated with the registry, which doesn't have a `host` equivalent. So, there's no way to make it consistent anyways.